### PR TITLE
feat(wix-ui-tpa): radio-button don't show focus ring on mouse click

### PR DESCRIPTION
**Pull request by urig-wix**
The desired improvement - changing the focus ring so it'd not appear after every mouse click (only from keyboard).

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_564/

File: src/components/RadioButton/docs/RadioButtonTestStory.tsx [hunk 1]
```diff
@@ -17,8 +16,6 @@ function renderTest(props?: any) {
           label="box theme"
           data-hook="radio-button-box"
         />
-      </div>
-      <div style={{ marginTop: '50px' }}>
```
Comment (jonathanadler) [inline src/components/RadioButton/docs/RadioButtonTestStory.tsx:21]: Let's leave the margins so the visual tests won't break 😁

**Review by jonathanadler — COMMENTED** [review]
Awesome!

Just one teeny tiny comment 😬

**Review by jonathanadler — APPROVED** [review]
Great, thanks!

**Pull request merged**